src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BaseComponent from 'lib/falcor/FalcorController';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPaths(params) {
+    return [
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['interactiveBySlug', params.slug, ['html', 'js', 'css']],
+    ];
+  }
+
+  render() {
+    const slug = this.props.params.articleSlug;
+    const reactHtml = {
+      __html: this.state.data.interactiveBySlug[slug].html,
+    };
+    return <div dangerouslySetInnerHTML={reactHtml} />;
```

Comment:
@zanemountcastle I just realized that doing it this way we still have our initial html `<head>` coming from the serverside render, so we probably don't need that in the boilerplate you wrote.

For other reasons I was also thinking that maybe we should just integrate the boilerplate repo you wrote into this component, then we can just use `React Helmet` etc. to change the head like in other places, yeah? This would also allow for us to change the layout of the boilerplate here and it would automatically apply to all older interactive articles which I think is quite good if for example we wanted to change the header for some UI reason.

What do you say?